lib/oelite/fetch/git.py

```diff
@@ -267,6 +267,17 @@ def unpack(self, d):
         return True
 
     def mirror(self, mirror=os.getcwd()):
+        if self.is_local:
+            # Don't mirror if the source is local. Concept taken from fetch
+            # Do check revision exists before accepting though
+            print "Warning: Skipping creating mirror for local path %s"%(self.uri)
```

Comment:
I understand your reasoning, but what do you want, no message at all? imo a warning is ok when you are trying to do something contradictory. Here you have instructed oe-lite to mirror something with a local url. Just silently ignoring it would likely end up with someone having to read through the source to figure out what is going on :) But, just make a dictator decision and let me know how you want it :)